src/query/expression/src/kernels/filter.rs

```diff
@@ -380,3 +314,95 @@ impl Column {
         new.into()
     }
 }
+
+impl FilterHelpers {
```

Comment:
It would be better to move FilterHelpers functions to `expresion` or other places? FilterHelpers is not bound to `kernels/filter`.